HierarchicalDataConverter/Catalyst.HierarchicalDataConverter/LoggingHelper2.cs

```diff
@@ -13,13 +13,14 @@ static LoggingHelper2()
             threadlock = new object();
         }
 
-        public static void Debug(Guid guid, string message)
+        public static void Debug(string message)
         {
             try
             {
                 lock (threadlock)
                 {
-                    LoggingHelper.Debug(message); // This is no longer working for some reason
+                    LoggingHelper.Debug(message); // Log to Engine Log
+                    // TODO - create plugin specific log
                     File.AppendAllText(
                         $@"C:\Program Files\Health Catalyst\Data-Processing Engine\logs\DataProcessingEngine_PluginOnly.log",
```

Comment:
Can you use another approach instead of hard-coding the path here (e.g. AppDomain.CurrentDomain.BaseDirectory)?

Reply:
Yeah - one of my next steps here is to add a logging framework which will clear out this ugly writing to file.  This was put in place to help isolate the logs (and also cause logs were not working for DPE at the time). I'll hopefully have a separate PR to get this cleaned up today or early tomorrow